adr/ADR-8.md

```diff
@@ -245,7 +247,17 @@ Here is a full example of the `CONFIGURATION` bucket:
   "num_replicas": 1,
   "duplicate_window": 120000000000,
   "rollup_hdrs": true,
-  "deny_delete": true
+  "deny_delete": true,
+  "allow_direct": true,
```

Comment:
At present this is default false.

Reply:
Ok so we may need to change the clients, GO had this https://github.com/nats-io/nats.go/blob/main/kv.go#L404
and I modified java/.net? @aricart

Reply:
So for now, I'm hearing that `allow_direct` should be false, but may change in the future.

Reply:
KV APIs should probably have a option that lets one enable it and later disable it.

Main concern are from being super new and not widely tested is using it requires updates to ACLs which in turn might also require a full redistribute of all client JWTs which might not be possible

Such flags will ensure it can be tested